### ISSUE TITLE
training data format?

### ISSUE THREAD

**Issue by lxj0276** [meijieru/crnn.pytorch#47]
is the img name is the label?can you show  me the training data format

**Comment by meijieru** [comment]
Please check readme more carefully, thanks
